public/controlpanel.php

```diff
@@ -43,6 +43,18 @@ function RenderUserPref(
 }
 ?>
 <script>
+function ShowLoadingIcon(iconRoot) {
+    $(iconRoot).find('.loadingicon-spinner').show();
+    $(iconRoot).find('.loadingicon-done').hide();
+    $(iconRoot).fadeTo(100, 1.0);
+}
+
+function ShowDoneIcon(iconRoot) {
+    $(iconRoot).find('.loadingicon-spinner').hide();
+    $(iconRoot).find('.loadingicon-done').show();
+    $(iconRoot).delay(750).fadeTo('slow', 0.0);
+}
```

Comment:
We are slowly phasing out jQuery.

This should be achievable with a combination of vanilla JS and TailwindCSS classnames.